multiversx_sdk_cli/sign_verify.py

```diff
@@ -48,3 +69,10 @@ def to_dictionary(self) -> Dict[str, str]:
 def sign_message(message: str, account: IAccount) -> SignedMessage:
     signature = account.sign_message(Message(message.encode()))
     return SignedMessage(account.address.to_bech32(), message, signature.hex())
+
+
+def sign_message_by_validator(message: str, validator: ValidatorSigner) -> SignedMessage:
+    message_computer = MessageComputer()
+    serialized_message = message_computer.compute_bytes_for_signing(Message(message.encode()))
```

Comment:
For validator wallets, this is not e regular use-case. In the few cases when validator wallets are required to sign a message, they sign the plain variant (raw bytes). Not this special enveloping:
 - https://github.com/multiversx/mx-sdk-py/blob/main/multiversx_sdk/core/message.py#L39

Reply:
As far as I know (see validators CLI of mxpy).

Reply:
Fixed